src/async-components/views/dialogs/security/CreateSecretStorageDialog.tsx

```diff
@@ -199,11 +191,8 @@ export default class CreateSecretStorageDialog extends React.PureComponent<IProp
                 // we may not have started crypto yet, in which case we definitely don't trust the backup
                 backupInfo ? await cli.getCrypto()?.isKeyBackupTrusted(backupInfo) : undefined;
 
-            const { forceReset } = this.props;
-            const phase = backupInfo && !forceReset ? Phase.Migrate : Phase.ChooseKeyPassphrase;
-
             this.setState({
-                phase,
+                phase: Phase.ChooseKeyPassphrase,
                 backupInfo,
```

Comment:
@richvdh fyi https://github.com/matrix-org/matrix-js-sdk/pull/4474

Reply:
Ah yes, ok. Rather than fetching the key backup info upfront and storing it in the `state`, can we do it just before calling `bootstrapSecretStorage` ? I think that will be simpler and clearer.